### ISSUE TITLE
Sorting rooms broken for virtual rooms

### ISSUE THREAD

**Issue by chagai95** [element-hq/element-web#20438]
#### What would you like to do?
When sorting based on activity also consider calls

#### Why would you like to do it?
I think it makes sense

### Have you considered any alternatives?

add a config for this if someone does not like it?

**Comment by SimonBrandner** [comment]
What behaviour do you have in mind exactly and how does it differ from the current one?

**Comment by SimonBrandner** [comment]
> What behaviour do you have in mind exactly and how does it differ from the current one?

@chagai95

**Comment by chagai95** [comment]
I still need to test this, It might be a case of virtual room events not being mapped. The behavior I had in mind was to move a room to the top of the list if it gets any call events, just like you would move it when it gets messages

**Comment by SimonBrandner** [comment]
Yeah, looks to be a bug with virtual rooms. Could you please rename the issue based on that?

**Comment by t3chguy** [comment]
Fixed by new room list
